hoomd/minimize/fire.py

```diff
@@ -1,217 +0,0 @@
-# coding: utf-8
```

Comment:
Remove the `hoomd/minimize` ignore lines from `setup.cfg` and `.pre-commit-config.yaml` as well.